week-4/Homework/mandatory/3-water-bottle.js

```diff
@@ -20,21 +20,31 @@ You have to implement the missing features according to the specification.
 // Here is your starting point:
 let bottle = {
   volume: 0,
-  fillUp: function() {
-    // calling this function should pour your bottle full (volume = 100);
+  fillUp() {
+    this.volume += 100; // calling this function should pour your bottle full (volume = 100);
   },

```

Comment:
Something is wrong here. Think about this part, if the volume is 80 already, what fill up will do ?

Reply:
fillup function modified to fill up to 100 in all cases.

Reply:
so what you wrote here in more description 
```this.volume= 100 -this.volume ;//this evaluated to
this.volume=this.volume+100-this.volume//right?
//so you can write in an easier way
this.volume=100;
```